docs/website/docs/developers/performance/profiling-with-tracy.md

```diff
@@ -51,6 +51,12 @@ Run IREE device binaries loading your modules | [Nothing particular](#running-th
 Run Tracy capture (`iree-tracy-capture`) to collect the trace | If device!=host (e.g. Android), [set up TCP port forwarding](#running-the-tracy-capture-cli-connecting-and-saving-profiles). | Same
 Build IREE's own tests and benchmark suites with Tracy instrumentation | [As above](#build-iree-device-binaries-with-tracy-instrumentation-clients), CMake: set `IREE_ENABLE_RUNTIME_TRACING`. | [Also need](#additional-steps-for-sampling) the CMake setting `IREE_BYTECODE_MODULE_FORCE_LLVM_SYSTEM_LINKER` so that `--iree-llvmcpu-link-embedded=false` will be passed to `iree-compile`.
 
+NOTE: If are using a `pip` release of IREE, it probably comes with a
+Tracy-enabled runtime already. You can select it by setting the
+`IREE_PY_RUNTIME=tracy` environment variable. Next time you run the IREE
+runtime, you should see the following message printed to stderr:
+'-- Using Tracy runtime (IREE_PY_RUNTIME=tracy)'.
```

Comment:
There is a already tip about `IREE_PY_RUNTIME` a bit further down on the page: https://iree.dev/developers/performance/profiling-with-tracy/#build-iree-device-binaries-with-tracy-instrumentation-clients

![image](https://github.com/openxla/iree/assets/4010439/cadb7792-168f-4e6b-a642-1051ae203ad0)

That tip is also using a mkdocs admonition/call-out (https://squidfunk.github.io/mkdocs-material/reference/admonitions/). How about moving the tip up from that section instead of duplicating the tip?

(I also think this whole page is due for a restructuring... there is a lot of content that takes some effort to parse through)

Reply:
What do you think about moving the exiting tip higher up the page then? I completely missed it on my first parse

Reply:
Yep, just moving the existing content up is fine with me.

Reply:
I reorganized this a bit PTAL.